packages/11ty/_layouts/README.md

```diff
@@ -48,4 +48,34 @@ A simple page template. Extends `base`.
 Raw JSON representation of publication data, used by search.
 
 #### splash
-Splash page layout. Extends `base`.
\ No newline at end of file
+Splash page layout. Extends `base`.
+
+### Page attributes
+#### Classes
+Classes added to the `class` frontmatter property of a layout or template are applied to the `<main>` element of that page.
+
+#### IDs
+The slugified page url is added as an id to the `<main>` element of each page.
+
+Example:
+
+```yaml
+// entry.liquid
+---
+class: quire-entry
+---
+```
+
+```yaml
+// catalogue/1.md
+---
+layout: entry
+class: fancy-custom-class
+---
+```
+
+```html
+// catalogue/1/index.html
+
+<main id="catalogue-1" class="quire-page quire-entry fancy-custom-class"></main>
+```
```

Comment:
```suggestion
A default `class` is set in the layout.

`_layouts/entry.liquid`
```yaml
---
class: quire-entry
---
```

Style classes can also be set in a template, these are appended to `class` value set in the layout, for example:

`content/catalogue/1.md`
```yaml
---
layout: entry
class: fancy-custom-class
---
```

`content/catalogue/1/index.html`
```html
<main id="catalogue-1" class="quire-page quire-entry fancy-custom-class">
  // omitted for example
</main>
```
```